compiler/rustc_codegen_llvm/src/llvm/mod.rs

```diff
@@ -229,16 +224,19 @@ pub(crate) fn set_thread_local_mode(global: &Value, mode: ThreadLocalMode) {
     }
 }
 
-impl AttributeKind {
+pub(crate) trait AttributeExt {
+    fn create_attr(self, llcx: &Context) -> &Attribute;
```

Comment:
Similarly, both of these `create_attr` methods could be inherent in `rustc_llvm`.